index.html

```diff
@@ -59,6 +59,130 @@ <h1><span class="greeting">Hey there. <br class="br"> I’m Denis.</span> <span
+          <h3 class="works__stories--heading">Multi-Post Stories<span class="gain-glory">Gain+Glory</span></h3>
+          <ul class="works__categories">
+            <li class="works__skill">Ruby on Rails</li>
+            <li class="works__skill">CSS</li>
+            <li class="works__skill">JavaScript</li>
+            <li class="works__skill">HTML</li>
+          </ul>
+          <a href="#" class="link see-project">See Project</a>
+        </div>
+      </article>
+      <article class="works__card">
+        <div class="information">
+          <h3 class="works__stories--heading">Multi-Post Stories<span class="gain-glory">Gain+Glory</span></h3>
+          <ul class="works__categories">
+            <li class="works__skill">Ruby on Rails</li>
+            <li class="works__skill">CSS</li>
+            <li class="works__skill">JavaScript</li>
+            <li class="works__skill">HTML</li>
+          </ul>
+          <a href="#" class="link see-project">See Project</a>
+        </div>
+      </article>
+    </div>
+  </section>
+
+  <section class="about-me">
+    <h2 class="about-me__heading">About me</h2>
+    <p class="statement">
+      I can help you build a product, feature or website Look through some of my work and experience! If you like what
+      you
+      see and have a project you need coded,<br class="br"> don’t hesitate to contact me.
+    </p>
+    <a href="#" class="link resume">Get my resume</a>
+
+    <div class="capabilities">
+      <article class="capability languages">
+        <img src="./img/about-me/languages.svg" alt="Image for languages" class="cap-img">
+        <h3 class="capability__heading">Languages</h3>
+        <ul class="capability__list languages__list">
+          <li class="item">JavaScript</li>
+          <li class="item">Ruby</li>
+          <li class="item">HTML</li>
+          <li class="item">CSS</li>
+        </ul>
+      </article>
+      <article class="capability frameworks">
+        <img src="./img/about-me/frameworks.svg" alt="Image for frameworks" class="cap-img">
+        <h3 class="capability__heading">Frameworks</h3>
+        <ul class="capability__list frameworks__list">
+          <li class="item">React</li>
+          <li class="item">Ruby on Rails</li>
+          <li class="item">RSpec</li>
+          <li class="item capybara">Capybara</li>
+          <li class="item selenium">Selenium</li>
+        </ul>
+      </article>
+      <article class="capability skills">
+        <img src="./img/about-me/skills.svg" alt="Image for skills" class="cap-img">
+        <h3 class="capability__heading">Skills</h3>
+        <ul class="capability__list skills__list">
+          <li class="item db-mgmt">Database Management</li>
+          <li class="item version-ctrl">Version Control</li>
+          <li class="item">CLI</li>
+          <li class="item">Web Development</li>
+          <li class="item">API Design</li>
+        </ul>
+      </article>
+    </div>
+  </section>
```

Comment:
- As per the project requirements, the technology categories ("Languages", "Frameworks" and "Skills") should be in a `ul`, each of them in a `li`. 
- The technology in each category(e.g., "JavasScript", "HTML" and "CSS") should also be a list `ul` and each of them a `li`

I would recommend replacing the `<div class="capabilities">` with a `ul`. That would make it a list and would make much more sense since it is actually a list of your skills.

- Each `article` should be a child of the main list and hence should be wrapped in a `li` tag. That would make each `article` a child of the main list.